lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<-EOM.gsub(/^ {6}/, '')
+
+      '#{METADATA_REPOSITORY}' repository not found.
+      Try running the command:
+        git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git "#{metadata_dir}"
```

Comment:
We could peek at their .git/config file and find out and report accordingly.  How their 'origin' reference is formed, we will know what their preference is.

It is just an extra push in being personally helpful, and is some information we really can peek at and know.

Some importance to being able to "copy and paste" a command was communicated, so if this is a valued, then there is value to being inclusive, not excluding those that  prefer another form.